package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "axios": "^0.21.1",
```

Comment:
You intsalled axios but I see no usage of it. Let's remove it

Reply:
Indeed, I was planning to use it but then found another way. Will be uninstalled with the next fix.